content_scripts/hints.js

```diff
@@ -42,6 +50,15 @@ var Hints = (function(mode) {
         holder = $('<div id=sk_hints/>');
     self.characters = 'asdfgqwertzxcvb';
 
+    function isMappedTo(keyPressed, keyToCheck) {
+        var mappingKeyPressed = Normal.mappings.find(encodeKeystroke(keyPressed));
+        var mappingKeyToCheck = Normal.mappings.find(encodeKeystroke(keyToCheck));
+
+        return mappingKeyPressed 
+            && mappingKeyPressed.meta
+            && mappingKeyPressed.meta.word === mappingKeyToCheck.meta.word;
```

Comment:
This line will raise error, when user users `unmap('j')`, as mappingKeyToCheck will be undefined in that case.